src/DBConnection.cpp

```diff
@@ -6,4 +6,8 @@ int MySqlDBConnection::query() const {
 
 Info MySqlDBConnection::advancedQuery() const {
 	return m_info;
+}
+
+int SqLiteDBConnection::query() const {
```

Comment:
Seems this already done in prev pr

Reply:
@malirod , no we don't have `SqLiteDBConnection` in prev pr. Only `SqLiteDBConnection2`. `SqLiteDBConnection` written and used in the first time

Reply:
Hm. So rename this class to sqliteconnetion3 - make naming consistent.

Reply:
@malirod , renamed and moved to `DBConnection3.hpp`